.github/workflows/tag.yml

```diff
@@ -0,0 +1,93 @@
+name: Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "The Infinispan branch to checkout when cutting the release."
+        required: true
+        default: "main"
+      version:
+        description: "Release version"
+        required: true
+      nextVersion:
+        description: "Next version"
+        required: true
+      dryRun:
+        description: "Dry run (doesn't push to Central)"
+        type: boolean
+        default: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '23'
```

Comment:
21?

Reply:
No, we need to compile with the latest JDK to support FFM.

Reply:
Ok, should we be running CI tests with 23 by default then?

Reply:
Yes, I think we should bump it up everywhere